deploy/docs/Best_Practices.md

```diff
@@ -1095,3 +1098,55 @@ The reliability of the system can be calculated using the following formulas:
 
 - If limited by queue_size: `number_of_instances*timeout[min]*sending_queue.queue_size/load_in_DPM` minutes.
 - If limited by PVC size: `number_of_instances*PVC_size/(1KB*load_in_DPM)` minutes.
+
+## Assigning Pod to particular Node
+
+### Using NodeSelectors
+
+Kubernetes offers a feature of assigning specific pod to node. Such kind of control is sometimes useful,
+whenever you want to ensure that pod will end up on specific node according your requirements like operating system
+or connected devices.
+
+### Binding pods to linux nodes
+
+Using this feature we can bind them to linux nodes.
+In order to do that `nodeSelector` has to be used. By default node selectors can be set for below pods:
+
+| component                   | key                                                                            | value |
+|-----------------------------|--------------------------------------------------------------------------------|-------|
+| `fluent-bit`                | `fluent-bit.nodeSelector.kubernetes.io/os`                                     | linux |
+| `fluentd`                   | `fluentd.events.statefulset.nodeSelector.kubernetes.io/os`                     | linux |
+| `fluentd`                   | `fluentd.logs.statefulset.nodeSelector.kubernetes.io/os`                       | linux |
+| `sumologic`                 | `sumologic.setup.job.nodeSelector.kubernetes.io/os`                            | linux |
+| `kube-prometheus-stack`     | `kube-prometheus-stack.prometheus-node-exporter.nodeSelector.kubernetes.io/os` | linux |
+| `kube-state-metrics`        | `kube-state-metrics.nodeSelector.kubernetes.io/os`                             | linux |
+| `prometheus`                | `prometheus.prometheusSpec.nodeSelector.kubernetes.io/os`                      | linux |
```

Comment:
I like the table :)

One thing though: I'd drop the `linux` column and move it to the section above as a lookup table for people that want to find a key for a particular component.